Neos.Flow/Classes/Mvc/Routing/Router.php

```
@@ -265,6 +267,14 @@ protected function createRoutesFromConfiguration()
                 }
                 $routesWithHttpMethodConstraints[$uriPattern] = false;
             }
+            if (isset($routeConfiguration['cache'])) {
+                if (isset($routeConfiguration['cache']['lifetime']) && is_int($routeConfiguration['cache']['lifetime'])) {
+                    $route->setCacheLifetime(RouteLifetime::createFromInt($routeConfiguration['cache']['lifetime']));
+                }
+                if (isset($routeConfiguration['cache']['tags']) && is_array($routeConfiguration['cache']['tags'])) {
```

same here
```suggestion
                if (isset($routeConfiguration['cache']['tags'])) {
                    assert(is_array($routeConfiguration['cache']['tags']), 'cache.tags must be of type array');
```

---

"Assertions should not be used for normal runtime operations like input parameter checks. As a rule of thumb your code should always be able to work correctly if assertion checking is not activated." 

https://www.php.net/manual/en/function.assert.php 

Maybe i get something wrong but it would consider settings kind of an input parameter.

---

This was just meant as an example.
My point was not to silently ignore invalid configuration, you could replace the `assert` with an exception of course.